src/content/ja/2020/caching.md

```diff
@@ -0,0 +1,938 @@
+---
+#See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
+title: キャッシング
+description: 2020年版Web Almanacのキャッシングの章では、キャッシュコントロール、期限切れ、TTL、有効性、可変、set-cookies、AppCache、サービスワーカー、機会について説明しています。
+authors: [roryhewitt, raghuramakrishnan71]
+reviewers: [jzyang]
+analysts: [raghuramakrishnan71]
+editors: [bazzadp]
+translators: [ksakae]
+roryhewitt_bio: <a hreflang="en" href="https://www.akamai.com/">Akamai</a>のエンタープライズアーキテクトで、パフォーマンスに情熱を注いでいます。イギリスからの帰国子女で、サンフランシスコに20年以上住んでいます。余暇には、長距離アドベンチャーバイク、スノーボーダー、ボクサー/カラテカとして活躍しています。彼はトラブルメーカーとして知られていることが好きです。そして何より、父親であり、夫であり、猫のルナの飼い主でもあるのです。
+raghuramakrishnan71_bio: <a hreflang="en" href="https://www.tcs.com/">Tata Consultancy Services</a>のエンタープライズアーキテクトで、公共部門の大規模なデジタルトランスフォーメーションプログラムに従事しています。テクノロジーに興味があり、特にパフォーマンスエンジニアリングに関心があります。旅行好きで、天文学、歴史、生物学、医学の進歩に興味を持っています。バガヴァッド・ギーターの第2章47節 "karmaṇy-evādhikāras te mā phaleṣhu kadāchana "を強く信奉している。
+discuss: 2056
+results: https://docs.google.com/spreadsheets/d/1fYmpSN3diOiFrscS75NsjfsrKXzxxhUMNcYSqXnQJQU/
+featured_quote: キャッシングは、コストのかかるネットワークリクエストを回避することで、パフォーマンスに大きなメリットをもたらします。これは、エンドユーザーにとっても（ウェブページを早く手に入れることができる）、ウェブページを提供する企業にとっても（サーバーの負荷を軽減する）メリットがあります。キャッシングはまさにWin-Winの関係なのです。
+featured_stat_1: 25.6%
+featured_stat_label_1: キャッシング情報のないHTTPレスポンス
+featured_stat_2: 21.4%
+featured_stat_label_2: 再検証ができない回答
+featured_stat_3: 21.3%
+featured_stat_label_3: キャッシングの改善により、再訪問時に2MB以上の節約が可能なサイト
+---
+
+## 序章
+
+キャッシングとは、一度ダウンロードしたコンテンツを再利用するための技術です。これは、何か（ウェブページを構築するサーバー、CDNなどのプロキシ、またはブラウザ自体）が「コンテンツ」（ウェブページ、CSS、JS、画像、フォントなど）を保存し適切なタグを付けることで再利用できるようにするものです。
+
+非常にハイレベルな例を挙げてみましょう。
+
+  *Janeは，ウェブサイトwww.example.comのホームページを訪れました．Janeはカリフォルニア州ロサンゼルスに住んでおり、example.comのサーバーはマサチューセッツ州ボストンにあります。Janeがwww.example.comにアクセスする際には、国を越えて移動しなければならないネットワークリクエストが発生します。*
```

Comment:
Looks like it's auto linking www.example.com so need to add some spaces to avoid this breaking:

```suggestion
  *Janeは，ウェブサイト www.example.com のホームページを訪れました．Janeはカリフォルニア州ロサンゼルスに住んでおり、example.comのサーバーはマサチューセッツ州ボストンにあります。Janeが www.example.com にアクセスする際には、国を越えて移動しなければならないネットワークリクエストが発生します。*
```